include/eve/conditional.hpp

```diff
@@ -38,22 +38,17 @@ namespace eve
 
   namespace detail
   {
+    template <logical_simd_value Logical>
+    Logical to_logical(eve::top_bits<Logical> mmask);
+
     template<typename T, relative_conditional_expr C>
```

Comment:
Shouldn't it just be an overload of to_logical over relative_conditional_expr ?

Reply:
`to_logical` for `relative_conditional_expr` is called `mask` and it's a member function.

In its implementation it uses top_bits, same as before.

Reply:
OK